src/app/app.component.html

```diff
@@ -0,0 +1,534 @@
+<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
```

Comment:
remove everything in this file. It's all generated code you dont need. Throw <h1>{title}</h1> instead